src/jit-analyze/jit-analyze.cs

```diff
@@ -125,14 +138,14 @@ public class MethodInfo
         {
             public string name;
             public int totalBytes;
-            public int prologBytes;
+            public double perfScore;
             public int functionCount;
             public IEnumerable<int> functionOffsets;
             public override string ToString()
             {
                 return String.Format(@"name {0}, total bytes {1}, prolog bytes {2}, "
                     + "function count {3}, offsets {4}",
-                    name, totalBytes, prologBytes, functionCount,
```

Comment:
If it does indeed make sense to remove prologBytes, you need to change the string "prolog bytes".

Reply:
ok